inference-engine/src/low_precision_transformations/src/fake_quantize_decomposition.cpp

```diff
@@ -118,18 +118,32 @@ DataPrecision getDataPrecisionByOutputPort(std::shared_ptr<opset1::FakeQuantize>
     }
 
     const auto& precisions = precisionsAttribute->get()->sharedValue->precisions;
+    std::vector<element::Type> precisionsForLevels{};
+    switch (levels) {
+        case 65536:
```

Comment:
minor, question: there are a lot of places where level values are used. Can we declare and use static constant anywhere?